config/config.go

```diff
@@ -181,13 +181,18 @@ var (
 		HTTPClientConfig: config.DefaultHTTPClientConfig,
 	}
 
+	DefaultRemoteWriteHTTPClientConfig = config.HTTPClientConfig{
```

Comment:
maybe we could continue using `config.DefaultHTTPClientConfig` as a basis and override the bool  `EnableHTTP2` or have some sort of embedding... to avoid diverging from the common defaults.